clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -2589,6 +2557,54 @@ public void testConsumerRejoinAfterRebalance() {
         }
     }
 
+    @Test
+    public void testThrowOnUnsupportedStableFlag() {
+        supportStableFlag((short) 6, true);
+    }
+
+    @Test
+    public void testNoThrowWhenStableFlagIsSupported() {
+        supportStableFlag((short) 7, false);
```

Comment:
Should we test with `true` ? If it is supported, we should not throw even if throwing is enabled? If yes, we can actually remove the parameter as we would pass in `true` for both cases.

Reply:
Never mind... the parameter has a different meaning. My bad.